src/Lykke.SlackNotification.AzureQueue/SlackMessageQueueContract.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Lykke.SlackNotification.AzureQueue
+{
+    public class SlackMessageQueueContract
```

Comment:
If it is a public shared type please add a description, if not please make it internal. The same for all new added types.